### DISCUSSION TITLE
[Question]ERROR while installing llvm.10.0.0

### DISCUSSION THREAD

**Discussion by robotjoaa** [prosyslab-classroom/cs348-information-security#482] [Q&A]
I was just setting up the environment according to the guideline. 
I followed everything up to the last line 

opam install dune llvm.10.0.0 ounit

and this happens. Other dune and ounit were installed.

![image](https://user-images.githubusercontent.com/36977804/110417564-2875cf00-80d9-11eb-9d03-ffb74e991714.png)

I wonder what's the problem.

**Comment by KihongHeo** [comment]
Hi, I am not sure this is the case or not but `[CS448]-4.11.0` is somewhat suspicious. My intention was `CS448-4.11.0`. I am wondering `[` makes the problem.

**Comment by robotjoaa** [comment]
starting again from the
opam switch create CS448-4.11.0 ... (without []) worked
